python/llm/test/benchmark/whisper/README.md

```diff
@@ -0,0 +1,35 @@
+# Whisper Test
+The Whisper Test allows users to evaluate the performance and accuracy of Whisper speech-to-text models.
+For accuracy, the model is tested on the [LibriSpeech] (https://huggingface.co/datasets/librispeech_asr) dataset using [Word Error Rate (WER)](https://github.com/huggingface/evaluate/tree/main/metrics/wer) metric.
+Before running, make sure to have [bigdl-llm](../../../README.md) installed.
+
+## Install Dependencies
+```bash
+pip install datasets evaluate soundfile librosa jiwer
+```
+
+## Run
+The LibriSpeech dataset contains 'clean' and 'other' splits. By default, the evaluation is done on the 'other' split.
+
+### Evaluation on CPU
```

Comment:
Maybe we can unify these two modules into one command, or put them in a shell script file. In fact, some environment variables may need to be set when running on the GPU, but we can make no distinction here. Taking CPU as an example, we will explain the meaning of the `device` field and let the user choose by themselves.